phalcon/Session/Manager.zep

```diff
@@ -323,6 +327,16 @@ class Manager extends AbstractInjectionAware implements ManagerInterface
             throw new Exception("The session adapter is not valid");
         }
 
+        /**
+         * Verify that the session value is alphanumeric, otherwise we
+         * unset the cookie to allow it to be created by session_start().
+         */
+        if fetch value, _COOKIE[name] {
+            if !ctype_alnum(value) {
```

Comment:
@davidofferman Can we change this to `preg_match("/^[a-z0-9]+$/iD", value)` which does the same thing and does not rely on `ctype`? I don't want to introduce a new requirement for Phalcon to run.